.github/workflows/unit-tests.yml

```
@@ -0,0 +1,20 @@
+name: Run unit tests
+
+# https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#on
+on:
+  pull_request:
+
+  push:
+    branches:
+      - develop
```

```suggestion
      - develop
      - staging
```
Also run tests on PRs into staging

---

👍🏻